src/Models/BaseElement.php

```diff
@@ -80,6 +80,10 @@ class BaseElement extends DataObject
         'BlockSchema' => DBObjectType::class,
         'IsLiveVersion' => DBBoolean::class,
         'IsPublished' => DBBoolean::class,
+        'canCreate' => DBBoolean::class,
```

Comment:
Adding to a private static would be considered a semver addition so this will need to be branched off and target `4` instead of `4.4`

Reply:
Ah had targeted 4.4 assuming this would count as a bug fix. But if there are specific rules about adding to private static needed to bump semver then can target 4

Reply:
> But if there are specific rules about adding to private static

Hey @scott1702 here's some more information on this:
* [Silverstripe CMS public APIs](https://docs.silverstripe.org/en/4/contributing/code/#picking-the-right-version) include private static class properties; and 
* According to SEMVER, a minor release "[MUST be incremented if new, backwards compatible functionality is introduced to the public API.](https://semver.org/#spec-item-7)"

Reply:
Perfect, thanks @brynwhyman!